src/num/gpuops.c

```diff
@@ -42,6 +43,24 @@ static void cuda_error(int line, cudaError_t code)
 	error("cuda error: %d %s \n", line, err_str);
 }
 
+static void printMemInfo()
+{
```

Comment:
- hm. this is a static function not used anywhere?
- please fix coding style (no camel case, spaces...=

Reply:
- I originally used the function only in `cuda_malloc`, but now it can be called from everywhere
- One might think of putting a call in `cuda_malloc` using a certain debug_level?!
- (Hopefully) fixed coding style